kge/model/kge_model.py

```diff
@@ -426,11 +458,62 @@ def create_from(
             if not config.log_folder or not os.path.exists(config.log_folder):
                 config.log_folder = "."
         dataset = Dataset.create_from(checkpoint, config, dataset, preload_data=False)
+
+        # do not load pretrained embeddings on creation from checkpoint
+        entity_pretrain_filename = config.get_default(
+            config.get("model") + ".entity_embedder.pretrain.model_filename"
+        )
+        relation_pretrain_filename = config.get_default(
+            config.get("model") + ".relation_embedder.pretrain.model_filename"
+        )
+        config.set(config.get("model") + ".entity_embedder.pretrain.model_filename", "")
+        config.set(
+            config.get("model") + ".relation_embedder.pretrain.model_filename", ""
+        )
```

Comment:
Why not pass a "init_for_load_only" argument to create() below and subsequently to all constructors? Then you can omit model-specific things not needed for loading without hard-coding it here.